Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -258,14 +272,15 @@ If the v2 address type is present in the `ceph quorum_status`, then the output o
 If you have multiple K8s clusters running, and want to use the local `rook-ceph` cluster as the central storage,
 you can export the settings from this cluster with the following steps.
 
-1) Copy create-external-cluster-resources.py into the directory `/etc/ceph/` of the toolbox.
+1. Copy create-external-cluster-resources.py into the directory `/etc/ceph/` of the toolbox.
 
-   ```console
-   toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
-   kubectl -n rook-ceph cp deploy/examples/create-external-cluster-resources.py $toolbox:/etc/ceph
-   ```
+    ```console
+    toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
+    kubectl -n rook-ceph cp deploy/examples/external/create-external-cluster-resources.py $toolbox:/etc/ceph
+    ```
 
-2) Exec to the toolbox pod and execute create-external-cluster-resources.py with needed options to create required [users and keys](#supported-features).
+2. Exec to the toolbox pod and execute create-external-cluster-resources.py with needed options to create required users and keys.
```

Comment:
dont remove the link

Reply:
This is a broken link, doesn't direct to anyting

Reply:
it actually working for me, and taking me to the top of the same document

Reply:
Use this `(#1-create-all-users-and-keys).`

Reply:
This works for me